src/module/actor/dataModelClasses/OseDataModelCharacterScores.js

```diff
@@ -0,0 +1,247 @@
+/**
+ * A class representing a character's ability scores
+ */
+export default class OseDataModelCharacterScores {
+  static standardAttributeMods = {
```

Comment:
It'd be nice to know in doc what attributes this and `cappedAttributeMods` applies to.

Reply:
Sure. I'll add it.